qsirecon/workflows/recon/anatomical.py

```diff
@@ -650,98 +578,86 @@ def _get_status():
         )
         odf_rois.inputs.input_image = crossing_rois_file
         workflow.connect([
-            (_get_source_node('t1_2_mni_reverse_transform'), odf_rois, [
-                ('t1_2_mni_reverse_transform', 'transforms')]),
-            (inputnode, odf_rois, [
-                ("dwi_file", "reference_image")]),
-            (odf_rois, buffernode, [("output_image", "odf_rois")])
+            (_get_source_node("template_to_anat_xfm"), odf_rois, [
+                ("template_to_anat_xfm", "transforms"),
+            ]),
+            (inputnode, odf_rois, [("dwi_file", "reference_image")]),
+            (odf_rois, buffernode, [("output_image", "odf_rois")]),
         ])  # fmt:skip
 
         # Similarly, if we need atlases, transform them into DWI space
-        if atlas_names:
+        if atlas_configs:
             desc += (
                 "Cortical parcellations were mapped from template space to DWIS "
                 "using the T1w-based spatial normalization. "
             )
 
             # Resample all atlases to dwi_file's resolution
-            get_atlases = pe.Node(
-                GetConnectivityAtlases(atlas_names=atlas_names),
-                name="get_atlases",
+            prepare_atlases = pe.Node(
+                WarpConnectivityAtlases(atlas_configs=atlas_configs),
+                name="prepare_atlases",
                 run_without_submitting=True,
             )
             workflow.connect([
-                (_get_source_node('t1_2_mni_reverse_transform'), get_atlases, [
-                    ('t1_2_mni_reverse_transform', 'forward_transform')]),
-                (get_atlases, buffernode, [
-                    ("atlas_configs", "atlas_configs")]),
-                (inputnode, get_atlases, [
-                    ('dwi_file', 'reference_image')])
+                (inputnode, prepare_atlases, [("dwi_file", "reference_image")]),
+                (prepare_atlases, buffernode, [("atlas_configs", "atlas_configs")]),
             ])  # fmt:skip
 
-        for atlas in atlas_names:
+        for atlas in atlas_configs.keys():
+            ds_atlas = pe.Node(
+                DerivativesDataSink(
```

Comment:
Do these not get overwritten by each other during the loop?

Reply:
The variables aren't used outside the loop and the node names include the atlas in them so it should be fine.